src/components/Courses/Courses.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import CourseCard from './components/CourseCard/CourseCard';
+import { buildAuthor } from './components/CourseCard/Author.types';
+import EmptyCourseList from './components/EmptyCourseList/EmptyCourseList';
+import './Courses.css';
+import Button from 'src/common/Button/Button';
+import SearchBar from './components/SearchBar/SearchBar';
+import { CoursesProp } from './Courses.types';
+
+const Courses: React.FC<CoursesProp> = (coursesProp) => {
+	if (coursesProp.courses.length > 0) {
+		const courseList = coursesProp.courses.map((course) => (
+			<CourseCard
+				id={course.id}
```

Comment:
add key prop here, it is always needed if you create a list of some components

Reply:
fixed